### PR TITLE
tests: Add unit tests to detect upcoming browser changes

### PR DESCRIPTION
**Pull request by bayotop**

### CODE REVIEW

**Review by xnerhu — APPROVED** [review]

**Comment by fpkamp** [comment]
@bayotop the schedule will conflict with other the agent jobs (17 and 18), so one is always gonna get cancelled, may I suggest scheduling at 17:30? the job that starts at 17 should have ended by then, but the job at 18 will take a lot longer

**Comment by bayotop** [comment]
> @bayotop the schedule will conflict with other the agent jobs (17 and 18), so one is always gonna get cancelled, may I suggest scheduling at 17:30? the job that starts at 17 should have ended by then, but the job at 18 will take a lot longer

@fpkamp Makes sense, thanks! Updated.

File: .github/workflows/check_upcoming_browser_versions.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+# You need to add BrowserStack access keys in order for the tests to work. Get a BrowserStack automate account, then go
+# to the repository settings, the Secrets page and add secrets: BROWSERSTACK_USERNAME, BROWSERSTACK_ACCESS_KEY.
+# In order to receive notifications in Slack, create a Slack bot,
+# add the credentials to repository secrets SLACK_BOT_TOKEN (when installing the app) and SLACK_CHANNEL_ID (from within Slack).
+
+name: Check upcoming browser versions
+
+on: 
+  schedule:
+    - cron: '30 17 ? * MON-FRI'
```
Comment (Finesse) [inline .github/workflows/check_upcoming_browser_versions.yml:11]: Let's use `0 19 ? * MON-FRI` to have a bigger gap between the scheduled pipelines

Comment (bayotop) [reply .github/workflows/check_upcoming_browser_versions.yml]: Updated.

**Review by r-valitov — APPROVED** [review]

**Pull request merged**